### ISSUE TITLE
Create search engine amongst employees

### ISSUE THREAD

**Issue by mariellelilja** [mariellelilja/IntraHub#46]
- Search bar
- Search page
- Hits page

To be able to perform search based on Employee's attributes such as (see all on #45 ) :
- Name
- Skillset tags
- Location, etc

Additionally (might be for another iteration/ another issue):
- Enable combined searches
- Enable filtering among hits
- Enable filtering in full Employee DB, without specific search

**Comment by mariellelilja** [comment]
Dependent on #45
